src/webpack.config.js

```diff
@@ -35,8 +35,8 @@ module.exports = {
 			{ test: /\.js?$/, loader: 'umd-compat-loader' },
 			{ test: /\.html$/, loader: 'html' },
 			{ test: /\.(jpe|jpg|png|woff|woff2|eot|ttf|svg)(\?.*$|$)/, loader: 'file?name=[path][name].[hash:6].[ext]' },
-			{ test: /\.styl$/, loader: ExtractTextPlugin.extract(['css-loader?sourceMap', 'stylus-loader']) },
-			{ test: /\.css$/, loader: 'style-loader!css-loader?modules' },
+			{ test: /\.styl$/, exclude: /\.module\.styl$/, loader: ExtractTextPlugin.extract(['css-loader?sourceMap', 'stylus-loader']) },
+			{ test: /\.module\.styl$/, loader: 'style-loader!css-loader?modules!stylus-loader' }
```

Comment:
I believe that we should be using `ExtractTextPlugin` here as well. Plus can we turn sourcemaps on.